cmd/controller-manager/app/controllermanager.go

```diff
@@ -146,7 +146,10 @@ func setupControllers(mgr controllerruntime.Manager, opts *options.Options, stop
 		Client:                       mgr.GetClient(),
 		EventRecorder:                mgr.GetEventRecorderFor(status.ControllerName),
 		PredicateFunc:                clusterPredicateFunc,
+		InformerManager:              informermanager.NewMultiClusterInformerManager(),
```

Comment:
Maybe we can share the `informerManager` between all controllers that will save the memory cost significantly.

We have initialized one at [here](https://github.com/karmada-io/karmada/blob/06e3bb360b4214ded4c6f8b2d3cfd894529a59a1/cmd/controller-manager/app/controllermanager.go#L213).

Reply:
Good idea. We could fix it in the next PR.